scripts/go-test-all.py

```diff
@@ -18,25 +18,34 @@ def require_env_var(name):
     return value
 
 def find_go_modules(directory):
-    """ Find all go.mod files in the current directory and subdirectories. """
     go_mod_files = []
     for root, _, files in os.walk(directory):
         if 'go.mod' in files:
             go_mod_files.append(root)
     return go_mod_files
 
+def has_test_files(module):
```

Comment:
had to change this to get tests working for `make test-all` locally otherwise it will fail on packages that have no tests 
```
no packages to test
make: *** [run-tests] Error 1
```